src/encoded/static/components/experiment_series.js

```diff
@@ -0,0 +1,356 @@
+                <h4 className="experiment-table__title">{`Experiments in experiment series ${context.accession}`}</h4>
+                <CartAddAllElements elements={experimentList} />
+            </div>
+        );
+
+        // Collect unique internal_tags from all relevant experiments.
+        internalTags = _.uniq(experimentList.reduce((allInternalTags, experiment) => (
+            experiment.internal_tags && experiment.internal_tags.length > 0 ? allInternalTags.concat(experiment.internal_tags) : allInternalTags
+        ), []));
+    }
+
+    return (
+        <div className={itemClass}>
+            <header className="row">
+                <div className="col-sm-12">
+                    <Breadcrumbs crumbs={crumbs} crumbsReleased={crumbsReleased} />
+                    <h2>Summary for experiment series {context.accession}</h2>
+                    {props.auditIndicators(context.audit, 'series-audit', { session: reactContext.session })}
+                    <DisplayAsJson />
+                </div>
+            </header>
+            {props.auditDetail(context.audit, 'series-audit', { session: reactContext.session, except: context['@id'] })}
+            <Panel addClasses="data-display">
+                <PanelBody addClasses="panel-body-with-header">
+                    <div className="flexrow">
+                        <div className="flexcol-sm-6">
+                            <div className="flexcol-heading experiment-heading"><h4>Summary</h4></div>
+                            <dl className="key-value">
+                                <div data-test="status">
+                                    <dt>Status</dt>
+                                    <dd><Status item={context} inline /></dd>
+                                </div>
+
+                                {context.description ?
+                                    <div data-test="description">
+                                        <dt>Description</dt>
+                                        <dd>{context.description}</dd>
+                                    </div>
+                                : null}
+
+                                {context.assay_term_name && context.assay_term_name.length > 0 ?
+                                    <div data-test="description">
+                                        <dt>Assay</dt>
+                                        <dd>{context.assay_term_name.join(', ')}</dd>
+                                    </div>
+                                : null}
+
+                                {terms.length > 0 || speciesRender ?
+                                    <div data-test="biosamplesummary">
+                                        <dt>Biosample summary</dt>
+                                        <dd>
+                                            {terms.length > 0 ? <span>{terms.join(' and ')} </span> : null}
+                                            {speciesRender ? <span>({speciesRender})</span> : null}
```

Comment:
I'm not sure that this wrapper `span` is necessary, since the component itself is wrapped in a `span` but also it is fine and then it matches the syntax above. Just a thought because you end up with a lot of nested `span`s.

Reply:
Not sure how to avoid the wrapper `<span>`. I changed it to:

```
{speciesRender ? { speciesRender } : null}
```

This was the only syntax I could make that would avoid an outright ESLint error, but then I crashed the server with a run-time error.